example/lib/main.dart

```diff
@@ -115,12 +132,16 @@ class _MyAppState extends State<MyApp> {
                       frequency: Duration(hours: 1),
                     );
                   }),
-              Text("iOS", style: Theme.of(context).textTheme.headline),
+              SizedBox(height: 16),
+              Text("Background Fetch (iOS only)", style: Theme.of(context).textTheme.headline),
```

Comment:
I would just get rid of it tbh.
I does not really belong in this library.
As an implementer I would just wanna know what is needed to use this library not how to debug it

Reply:
I tend to disagree ; without this button (which is more a test than a debug button), there's no way for the plugin user to get his own code called, except for waiting for iOS to actually trigger `performFetchWithCompletionHandler` (which may take days).
Also, we should either provide a demo (/test) sample for both platforms or for none, not halfways :)

Reply:
It's the same for Android.
You will need to wait.
We don't even have the ability to forcefully run the job even in debug mode.

I think the iOS code could stay in the sample but just not provided to the enduser.

The enduser can expect a fully working plugin from us